README.rst

```diff
@@ -51,8 +51,8 @@ Getting Started
 To use PyPIM, you must have access to the PIM API.
 
 .. note::
-    The PIM API is a work in progress. There is no public exposure or
-    distribution of an implementation yet.
+    The PIM API is a work in progress. Even though the API definition and the
+    PyPIM client are published, the service itself is not publicly exposed yet.
```

Comment:
You say “exposed yet”, does that mean you have. Pune to make the service implementation public?

Reply:
Nothing public, answering in private